### PR TITLE
operator entry

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

File: kogito-travel-agency/operator/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+kubectl wait --for=condition=ready pod  --selector=app=infinispan-pod --timeout=5m
+kubectl wait --for=condition=ready pod  --selector=strimzi.io/kind=Kafka --timeout=5m
+```
+
+After Infra is ready let's deploy our Travel and Visa Service
+```shell-script
+ kubctlt apply -f travel-agency/ 
+```
+
+After the deployment just relax and sit back while operator take care of the rest.
```
Comment (manaswinidas) [inline kogito-travel-agency/operator/README.md:43]: take -> takes

**Review by manaswinidas — COMMENTED** [review]
Looks great! Thanks @Kaitou786

File: kogito-travel-agency/operator/README.md (hunk 1 above)
Comment (Kaitou786) [reply kogito-travel-agency/operator/README.md:43]: Can you mark it as suggestion and commit directly using GUI?

Comment (manaswinidas) [reply kogito-travel-agency/operator/README.md:43]: Ok by commit do you mean merge?

**Pull request merged**